src/root/stylesheets/variables.scss

```diff
@@ -2,6 +2,12 @@
 
 $content-bg-color: #f5f5f5;
 $primary-text-color: #6c6c6e;
+$primary-border-color: #dee2e6;
+
+$tab-active-color: #f8f8f8;
```

Comment:
I think it would make more sense to keep component variables local to the component. It's impossible to know which tab component are we talking about here, there two so far in the app.

Reply:
Hmmm... I think having a centralized variables file for the whole project does make more sense. Maybe we should implement a more specific wording or a less specific wording of the variables to enable a global use.
For future theming the solution of keeping the variables local to the component doesn't look right to me.

I'm not talking about a setting nightmare like foundation has, but a colors and sizes group of variables is something you should consider.

Reply:
For what reason doesn't it look right? I don't really see sizings / spacings and style specifics of a single component as being suitable for global use. I can see them being potentially misused. If correctly decoupled, each component should only worry about its own styles.

Common colors, states, font stylings, spacings, form outlines etc. would be useful to all components. 

I think the challenge is to provide a set of common variables that are minimal and obvious but useful to all components.

Reply:
"Common colors, states, font stylings, spacings, form outlines etc. would be useful to all components." 
We are talking about $tab-active-color and such, which (renamed) are usefull in a global scope.

The sizes variables are also logical in a global scope (for example the $menu-width), because the last commit to the issue showed that a "magical number" as you called it, can cause confusion.

What I'm suggesting is to rename the tab-colors to match their color: for example add a $color-light, $color-medium and a $color-dark and then use them inside the components or reasign them to other component specific variables.

But redeclaring the menu-width variable (for example) in all needed files (menu.scss and tabs.scss) doesn't look an feel right to me.

Reply:
bq. We are talking about $tab-active-color and such, which (renamed) are usefull in a global scope.
Exactly, the active/selected color is useful in a global scope, but it wouldn't name it specific to tabs.

bq. The sizes variables are also logical in a global scope (for example the $menu-width), 
I see the menu width as local to the authenticated layout component. No other component should need to be concerned with it if it's handled there.

bq. What I'm suggesting is to rename the tab-colors to match their color: for example add a $color-light, $color-medium and a $color-dark
Right, these component colors could be reused and defined globally.

I think what we're missing is a global spec for all these colors and states, but I guess that will come out of the design discussions that are happening around the nOS design competition.

Reply:
@mmoshi I agree! Would you have a look at the last commit and tell me what you think of it?

Reply:
Sure, I can add some more comments, but perhaps it would be a good idea to check with the team whether they want to commit to this design now before we we spend time making any more adjustments. Might be better to consider a redesign of the entire browsing area (bar, tabs, etc.) before doing this to see how it would fit in.